code/renderers/svelte/src/components/SlotDecorator.svelte

```diff
@@ -1,16 +1,11 @@
 <script>
-  import { onMount } from 'svelte';
   import { VERSION } from 'svelte/compiler';
-
   export let decorator = undefined;
```

Comment:
let's convert these to the `$props()` rune too

Reply:
Can you please try it? I had a long fourth and back with transforming `SlotDecorator` and other files to runes, and for some reason, components weren't rendered anymore. Therefore, [I've reverted the change](https://github.com/storybookjs/storybook/pull/30703/commits/d610be08b11e6afd065d03e64e8054a843e9350c).